tuist/acho
# Updating Swift package

### Short description 📝

Attempting to use `acho` from other swift packages was failing 

### Solution 📦

- Exposing `acho` as a library (to allow other packages to leverage it) within the swift package 

### Notes

- Also updating readme with correct version number

### Test Plan 🛠

- Create a dummy package
- Add acho as a dependency within `Package.swift`
- Verify the package successfully builds!
